app/src/main/res/values-pl/strings.xml

```diff
@@ -312,11 +310,34 @@
     <string name="send_toot_notification_channel_name">Wysyłanie wpisów</string>
     <string name="send_toot_notification_cancel_title">Anulowano wysyłanie</string>
     <string name="send_toot_notification_saved_content">Kopia wpisu została zapisana jako szkic</string> 
-
-    <string name="download_image">Pobieranie %1$s…</string>
-
-    <string name="action_copy_link">Skopiuj odnośnik</string>
-    <string name="status_sent">Wysłano!</string>
-    <string name="status_sent_long">Pomyślnie wysłano odpowiedź.</string>
+    <string name="action_compose_shortcut">Wyślij</string>
+
+    <string name="error_no_custom_emojis">Twoja instancja %s nie używa żadnych niestandardowych emoji</string>
+    <string name="copy_to_clipboard_success">Skopiowano do schowka</string>
+    <string name="emoji_style">Styl emoji</string>
+    <string name="system_default">Domyślny systemu</string>
+    <string name="download_fonts">Musisz najpierw pobrać te zestawy emoji</string>
+    <string name="performing_lookup_title">Wyszukiwanie…</string>
+    <string name="expand_collapse_all_statuses">Rozwiń/zwiń wszystkie wpisy</string>
+    <string name="action_open_toot">Otwórz wpis</string>
+    <string name="restart_required">Wymagane jest ponowne uruchomienie</string>
+    <string name="restart_emoji">Musisz uruchomić ponownie Tusky-ego, aby zastosować zmiany</string>
```

Comment:
In this case you should not use neither dash nor apostrophe. Source: https://sjp.pwn.pl/poradnia/haslo/Apostrof-i-lacznik-w-odmianie-wyrazow;18398.html